pkg/client/restclient/request.go

```diff
@@ -339,16 +339,16 @@ func (r resourceTypeToFieldMapping) filterField(resourceType, field, value strin
 
 type versionToResourceToFieldMapping map[unversioned.GroupVersion]resourceTypeToFieldMapping
 
+// filterField transforms the given field/value selector for the given groupVersion and resource
 func (v versionToResourceToFieldMapping) filterField(groupVersion *unversioned.GroupVersion, resourceType, field, value string) (newField, newValue string, err error) {
 	rMapping, ok := v[*groupVersion]
 	if !ok {
-		glog.Warningf("Field selector: %v - %v - %v - %v: need to check if this is versioned correctly.", groupVersion, resourceType, field, value)
```

Comment:
The warning is complaining about selectors that work today... dozens of objects support `metadata.name` filtering, and this complains because we haven't defined client-side "metadata.name":"metadata.name" for all those objects. If you try to use a field selector that doesn't work, the server complains.